template/config/hello-world/hello-world.ts

```diff
@@ -1,6 +1,7 @@
-import { Component, Vue } from 'vue-property-decorator';
+import { Vue } from 'vue-property-decorator';
+import { StoryComponent } from '.storybook/modules';
 
-@Component({
+@StoryComponent({
```

Comment:
This is a page; it should use @Component, not @StoryComponent. Sorry if i told you otherwise. Pages do not show up in storybook and so don't use that decorator.

Reply:
this has been corrected.